app.js

```diff
@@ -91,25 +91,24 @@ app.use(function (req, res, next) {
 });
 
 // error handler
-app.use(function (err, req, res, next) {
-    // set locals, only providing error in development
-    let status = err.status || err.statusCode;
-    if (err.statusCode) {
-        res.setHeader("error-message", err.error.message);
-        res.locals.message = err.error.message;
-    }else {
-        res.locals.message = err.message;
-    }
-    res.locals.error = req.app.get('env') === 'development' ? err : {status};
-
-    if (res.locals.currentUser)
-        res.locals.loggedin = true;
-    // render the error page
-    res.status(status);
-    res.render('lib/error', {
-            loggedin: res.locals.loggedin,
-            inline: res.locals.inline ? true : !res.locals.loggedin
-        });
+app.use((err, req, res, next) => {
+	// set locals, only providing error in development
+	const status = err.status || err.statusCode || 500;
+	if (err.statusCode) {
+		res.setHeader('error-message', err.error.message);
```

Comment:
Würde ich nur im dev-mode anzeigen, da in den Fehlermeldungen kritische Informationen stecken könnten.